src/newItinerary/internals/Line.style.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components'
+
+import { color, space } from '../../_utils/branding'
+import { Lines } from '../Lines'
+
+export const StyledLineWrapper = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-height: 32px;
+  width: 10px;
+  margin-right: ${space.m};
+`
+
+// Using background-image to display the dashed line for connection
+const base64EncodedDash =
+  'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMCIgaGVpZ2h0PSIxMDAiPjxsaW5lIHgxPSIyIiB4Mj0iMiIgeTE9IjUiIHkyPSIxMDAlIiBzdHJva2U9IiM3MDhDOTEiIHN0cm9rZS13aWR0aD0iMyIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIiBzdHJva2UtZGFzaGFycmF5PSIuMDAxLCAxMCIvPjwvc3ZnPg=='
+
+const lineToBackground: Readonly<{ [key in Lines]: string }> = {
+  [Lines.NONE]: 'transparent',
+  [Lines.ACTIVE]: color.midnightGreen,
+  [Lines.INACTIVE]: color.gray,
+  [Lines.CONNECTION]: `repeat-y url("data:image/svg+xml;base64,${base64EncodedDash}")`,
+  [Lines.HIDDEN_STOPS]: color.midnightGreen,
+}
+
+export const StyledLine = styled.div<{ line: Lines }>`
+  width: 4px;
```

Comment:
could be part of a `border` token and so declared as variable to avoid magic numbers

Reply:
I'm not a fan with of factorizing things that are not (yet) in the design. Scoping this `width` here make sense to me, because it's not related to any other component.

Reply:
Put a `LINE_WIDTH` constant at the top of the file